### DISCUSSION TITLE
Discussion page turned on

### DISCUSSION THREAD

**Discussion by lofstrandbjorn** [AMSP-04/NETN-FOM#48] [Announcements]
We turned on "discussions" feature.

**Comment by cakelingkia** [comment]
Hi,

   I would like to find out what are the object/interaction classes in NETN-FOM Version 3.0 that can be used for representing the land slide and flooding. Base on my limited knowledge of the NETN-FOM and RPR FOM 2.0, these events and representation are currently not supported in the NETN-FOM. If these environmental effects are currently not represented yet, which is the module, object/interaction best suited to extend to support these kind of representation. 

   I would think the METOC module "HLAobjectRoot.METOC_Root.EnvironmentCondition.Weather.LandSurface" object class can extend an additional attribute call flood for supporting flood similar what was defined in Snow attribute. 

   In our project, our user would also like to visualize these effects in the simulated 3D world if these events are taken place as well.

   Thanks.

With Regards,

Chee Kiong

**Comment by lofstrandbjorn** [comment]
Chee,

The Landslide and Flooding object and interaction classes are in Version 4.0 of NETN-FOM and specifically in the NETN-DIM module.
NETN-FOM v4 is complete and scheduled for release by NSO in the upcoming months but you can download the complete set of modules from the NETN-FOM repo in the develop branch.

With Regards,

/Björn Löfstrand

**Comment by lofstrandbjorn** [comment]
Hi Bjorn,

   Thanks for your prompt reply and thanks.

   Have a nice day.

With Regards,

Chee Kiong

From: Björn Löfstrand ***@***.***>
Sent: Friday, May 10, 2024 3:47 PM
To: AMSP-04/NETN-FOM ***@***.***>; AMSP-04/NETN-FOM ***@***.***>; Ng Chee Kiong ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [AMSP-04/NETN-FOM] Discussion page turned on (Discussion #48)


*** WARNING! THIS EMAIL ORIGINATES FROM OUTSIDE ST ENGINEERING. ***
                  Do not click on links or attachments if you are unsure of the source.
Chee,

The Landslide and Flooding object and interaction classes are in Version 4.0 of NETN-FOM and specifically in the NETN-DIM module.
NETN-FOM v4 is complete and scheduled for release by NSO in the upcoming months but you can download the complete set of modules from the NETN-FOM repo in the develop branch.

With Regards,

/Björn Löfstrand
